index.html

```diff
@@ -897,6 +996,33 @@ <h1 style="font-size: 48px;">My headline</h1>
 
 						---
 
+						### Pseudo-classes
```

Comment:
Moved these up to be closer to content that they seem more similar to (selectors + text styles). I think the exercise should still work fine, not relying on anything from the positioning/layout section.